packages/kbn-pm/src/commands/bootstrap.ts

```diff
@@ -145,7 +145,11 @@ export const BootstrapCommand: ICommand = {
       upstreamBranch: kbn.kibanaProject.json.branch,
       // prevent loading @kbn/utils by passing null
       kibanaUuid: kbn.getUuid() || null,
-      timings: timings.map((t) => ({ group: 'scripts/kbn bootstrap', ...t })),
+      timings: timings.map((t) => ({
+        group: 'scripts/kbn bootstrap',
+        ...t,
+        meta: { nestedTiming: process.env.CI_STATS_NESTED_TIMING },
```

Comment:
Maybe we should include this in the default metadata?

Reply:
Good call.

Reply:
https://github.com/elastic/kibana/pull/113492/files#diff-5ad541ac975af33d224bd5f0a56b6e8d0c0c28337d51926a3c66dc45af5ac05bR106